docs/snippets/INDEX.md

```diff
@@ -169,26 +167,41 @@ _All snippets, organized by the [`Archetype`](https://rerun.io/docs/reference/ty
-| **[`Scalar`](https://rerun.io/docs/reference/types/archetypes/scalar)** | `archetypes⁠/⁠scalar_row_updates` | Update a scalar over time | [🐍](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/archetypes/scalar_row_updates.py) | [🦀](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/archetypes/scalar_row_updates.rs) | [🌊](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/archetypes/scalar_row_updates.cpp) |
-| **[`Scalar`](https://rerun.io/docs/reference/types/archetypes/scalar)** | `archetypes⁠/⁠scalar_multiple_plots` | Log a scalar over time | [🐍](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/archetypes/scalar_multiple_plots.py) | [🦀](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/archetypes/scalar_multiple_plots.rs) | [🌊](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/archetypes/scalar_multiple_plots.cpp) |
-| **[`Scalar`](https://rerun.io/docs/reference/types/archetypes/scalar)** | `archetypes⁠/⁠scalar_column_updates` | Update a scalar over time, in a single operation | [🐍](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/archetypes/scalar_column_updates.py) | [🦀](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/archetypes/scalar_column_updates.rs) | [🌊](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/archetypes/scalar_column_updates.cpp) |
-| **[`Scalar`](https://rerun.io/docs/reference/types/archetypes/scalar)** | `archetypes⁠/⁠series_line_style` | Log a scalar over time | [🐍](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/archetypes/series_line_style.py) | [🦀](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/archetypes/series_line_style.rs) | [🌊](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/archetypes/series_line_style.cpp) |
-| **[`Scalar`](https://rerun.io/docs/reference/types/archetypes/scalar)** | `archetypes⁠/⁠series_point_style` | Log a scalar over time | [🐍](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/archetypes/series_point_style.py) | [🦀](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/archetypes/series_point_style.rs) | [🌊](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/archetypes/series_point_style.cpp) |
+| **[`Scalar`](https://rerun.io/docs/reference/types/archetypes/scalar)** | `archetypes⁠/⁠scalars_simple` | Log a scalar over time | [🐍](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/archetypes/scalars_simple.py) | [🦀](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/archetypes/scalars_simple.rs) | [🌊](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/archetypes/scalars_simple.cpp) |
```

Comment:
I suggest we hide all deprecated types from the snippet INDEX.md, as well as from all docs (`#[doc(hidden)]`)

Reply:
Added an issue for it:
* https://github.com/rerun-io/rerun/issues/9371